Makefile

```diff
@@ -67,3 +72,32 @@ run-local-rookify: ## Runs rookify in the local development environment (require
 ROOKIFY_VERSION ?= 0.0.0.dev0
 build-container: ## Build container from Dockerfile, add e.g. ROOKIFY_VERSION=0.0.1 to specify the version. Default value is 0.0.0.dev0
 	${CONTAINERCMD} build --build-arg ROOKIFY_VERSION=$(ROOKIFY_VERSION) -t rookify:latest -f Dockerfile .
+
+.PHONY: run-container
+export ROOKIFY_VERSION ?= "0.0.0.dev0"
+run-container: ## Runs the container as specified in docker-compose.yml and opens a bash terminal
+	${CONTAINERCMD} compose up -d
+
+.PHONY: run-tests-locally
+run-tests-locally: ## Runs the tests in the tests directory. NB: check that your local setup is connected through vpn to the testbed!
+	$(eval PYTHONPATH="${PYTHONPATH}:$(pwd)/src") \
+	source ./.venv/bin/activate && \
```

Comment:
This can be implemented in an easier way. Each venv has it's own "binaries" under `/bin` (see `rookify` entrypoint in the `Dockerfile` for reference). You may just call `/app/rookify/.venv/bin/python`.